pkg/controller/scyllacluster/sync_services.go

```diff
@@ -360,13 +360,13 @@ func (scc *Controller) syncServices(
 
 				klog.V(2).InfoS("Pod has not been recreated by the StatefulSet controller yet",
 					"ScyllaCluster", klog.KObj(sc),
-					"Pod", klog.KObj(pod),
+					"Pod", klog.KObj(svc),
```

Comment:
I think `klog.ObjectRef{Namespace: svc.Namespace, Name: svc.Name}` would be more fitting

Reply:
why would be more fitting? KObj is essentially doing that.

Reply:
It's more explicit. Imo `"Pod", klog.KObj(svc)` looks unintentional and if I ever saw that without any comment, I'd probably think it's a mistake.